src/login/components/LoginFormWalletFile/LoginFormWalletFile.js

```diff
@@ -47,22 +52,59 @@ export default class LoginFormWalletFile extends React.Component {
   }
 
   renderAccounts = () => {
-    const { accounts, wif } = this.props;
+    const { accounts, encryptedWIF } = this.props;
 
     if (accounts.length === 0) {
       return null;
     }
 
     return (
-      <Select className={styles.accounts} value={wif} onChange={this.handleSelect}>
-        <option value="">Select an account</option>
-        {map(this.props.accounts, (account, index) => (
-          <option value={account.encrypted} key={`account${index}`}>{account.label}</option>
-        ))}
-      </Select>
+      <div>
+        <Select className={styles.accounts} value={encryptedWIF} onChange={this.handleSelect}>
```

Comment:
This admittedly was missing before you worked on it, but would you mind adding `disabled={disabled}` (based upon `this.props.disabled`) in case the login action is processing?